### PR TITLE
chore: update icons from latest version of Figma file

### PR DESCRIPTION
**Pull request by ju-Skinner**
# Description
Ran the Figma export process to get the latest icons.

### CODE REVIEW

File: libs/icons/changelogs/2023-08-22-changelog.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>2023-08-22 - {{version}} Changelog</title>
```
Comment (pixelflips) [inline libs/icons/changelogs/2023-08-22-changelog.html:5]: Not a blocker, but `{{version}}` outputs as text.

**Review by pixelflips — APPROVED** [review]
One tiny non-blocking comment but overall LGTM! 👍🏼 Great work!

**Review by anechol — APPROVED** [review]
🚀

File: libs/icons/changelogs/2023-08-22-changelog.html (hunk 1 above)
Comment (ju-Skinner) [reply libs/icons/changelogs/2023-08-22-changelog.html:5]: Yeah, that's still in the works. Which will happen during nightly builds.

**Pull request merged**